packages/create-angular/package.json

```diff
@@ -24,7 +24,7 @@
     "@angular/platform-browser-dynamic": "^18.2.13",
     "@angular/router": "^18.2.13",
     "@auth0/auth0-angular": "^2.2.3",
-    "@carto/api-client": "^0.4.6",
+    "@carto/api-client": "^0.5.0-alpha.3",
```

Comment:
No change needed now but just a "note to self" that we'll probably want to publish a stable version of the API client before merging this PR.